examples/pos/liftinference.effekt

```diff
@@ -5,7 +5,7 @@ interface Foo {
   def bar(): String
 }
 
-extern control def fancy { f: Unit => Int }: Int = "foo"
+extern control def fancy { f: Unit => Int }: Int = simpleAtom "foo"
```

Comment:
Here, using `default { ... }` instead would potentially make the test less useful - after all, there probably was a reason for using an extern here, and this will never reach `LiftInference` then (will be desugared in core already).

Reply:
Oh that is true. But we could also run it only in one of the lift backends.

Reply:
I'll probably just duplicate it. 🤔